keyboards/dz60/keymaps/olivierko/readme.md

```diff
@@ -0,0 +1,19 @@
+# DZ60
+
+### _BL:
+Base layer with american ANSI layout.
+![_BL](https://i.imgur.com/BPMn7dk.png)
+### _SL:
+Swedish layer with ÅÄÖ at original positions.
+![_SL](https://i.imgur.com/I8QRh24.png)
+### _FL:
+Function layer including various extra keys.
+![_FL](https://i.imgur.com/nCbCIrZ.png)
+### _CL:
+Control layer for managing RGB and flashing.
+![_CL](https://i.imgur.com/QnoMgsb.png)
+
+### Make command:
+```sh
+make dz60:olivierko:dfu
```

Comment:
QMK now has a `:flash` target that will use the correct program as long as the bootloader is set correctly.

```suggestion
make dz60:olivierko:flash
```